js/stores/appStore.js

```diff
@@ -239,7 +239,7 @@ function windowDefaults () {
     windowOffset: 20,
     webPreferences: {
       sharedWorker: true,
-      partition: ''
+      partition: 'main-1'
```

Comment:
to make everything work correctly with private tabs and extensions, we should have to identically named partitions with one in-memory and one persistent. The logic in electron takes care of matching them together. Private tabs should use the in-memory '' partition and the browser process should load favicons through the partition that matches the one used by the tab. Not a big deal right now because the extensions we are using don't really need the spanning support, but eventually it will be an issue